src/flick/lst/controllers/update_lst_controller.py

```diff
@@ -32,11 +33,14 @@ def _add_shows(self, show_ids):
                 show = Show.objects.get(pk=show_id)
                 self._lst.shows.add(show)
 
-    def _add_collaborators(self, collaborator_ids):
+    def _add_collaborators(self, collaborator_ids, request_user):
         self._lst.collaborators.clear()
         for c_id in collaborator_ids:
             if User.objects.filter(pk=c_id):
                 collaborator = User.objects.get(pk=c_id)
+                collaborator_friend = Friend.objects.get(to_user=request_user, from_user=collaborator)
```

Comment:
this might throw an error, you should always check with `filter` first before you do a `get`